src/GenerateProgrammaticPipelineFromConfig/Generator.php

```diff
@@ -241,32 +241,20 @@ private function generateRoutes(array $config)
                 $route = empty($spec['name'])
                     ? sprintf(self::TEMPLATE_ROUTED_NO_METHOD_NO_NAME, $path, $middleware)
                     : sprintf(self::TEMPLATE_ROUTED_NO_METHOD_WITH_NAME, $path, $middleware, $spec['name']);
-
-                goto options;
```

Comment:
I'm one of the folks who doesn't like `goto` :)

`if..elseif..else` statement doesn't produce more code there. There is no any duplication right now, as it was also with `goto` instruction. So basicaly now this 'if...else` statement is to get correct `route` - so I think it should be extracted to separate method, as I just did in 8088334.

Do you still think it is harder to understand and less maintainable?